gitax/Visual Studio Projects/C Sharp Projects/GTXLibGit2Sharp/GTXRepo.cs

```diff
@@ -20,8 +20,8 @@ public static class GTXRepo
         /// <param name="repoPath">the repository main path</param>
         public static void Init(string repoPath)
         {
-            if (!Repository.IsValid(repoPath))
-                Repository.Init(repoPath, false);
+            if (!Repository.IsValid(repoPath))            
+             Global.error(repoPath  + " is not a valid git repo");            
```

Comment:
Yeah. This is called once you start ax client and every time you go to VCS parameters and set a folder in there. If it does not make sense we might need to create a button to 'initialize' the git repository.

But as you clarified before, you are using only the sync (which I agree is not working properly yet). In your case when you setup VCS parameters folder you will have already a git directory...

Any other suggestions?